api/views/maps.py

```diff
@@ -0,0 +1,77 @@
+from api import app
+from flask import Blueprint, request
+from api import db
+from api.models import Map
+import json
+from flask import jsonify
+#from api.utils import InvalidUsage
+from sqlalchemy import func
+# import time
+# from datetime import date
+# import uuid
+from flask_sqlalchemy import SQLAlchemy
+from api.utils import create_response
+
+
+
+mod = Blueprint('maps', __name__)
+
+
+#none of these have been comprehsnsively tested yet
+
+@app.route('/maps', methods = ['GET'])
+# Get all map years.
+# get function is definitely not functional
+def get_map_years():
+    if request.method == 'GET':
+        data_in = to_dict(Map.query.all())
```

Comment:
you can only call `to_dict` on a single model instance. if you want to process a list, it should look something like [this](https://github.com/hack4impact-uiuc/nnb-backend-v2/blob/master/api/views/pois.py#L22)

Reply:
using the list comprehension form will also allow you to not use the `for` loop below